### DISCUSSION TITLE
Not found air conditioner Rotenso roni x after update 2024.7.0

### DISCUSSION THREAD

**Discussion by piotrek2555** [make-all/tuya-local#2144] [Q&A]
I update Tuya local to 2024.7.0 and i have problem with air conditioner Rotenso roni x.

Don`t found my device. I even try Delete my devices from configuration tuya, and add once again, with app tuya, I scan and sucessfully adding to my Home asistant. On elder version working all function my air conditioner. 

In type device I have only:

co2_box
daizuki_heatpump

In version 2024.6.0 is roniX device type, only in 2024.7.0 haven`t.
I have noticed you, that scale temeprature is to 16 to 88 Celsius degrees, my device support only 16 to 31 degrees Celsius. Fix it. 

But i pick fisher_summer_airconditioner, this scale is from 16 to 21 degrees and most function working.

**Comment by make-all** [comment]
It is possible you have been bitten by a suspected bug in HACS, where it prompts you to reboot too soon after updating an integration, and the contents do not get unpacked fully.  There have been several reports of this happening since around April, I am not sure if it was an update in HACS or an update in HA that caused this.
Otherwise, if the file is there in `custom_components/tuya_local/devices/`, there may be more info in the log that can be used to figure out why it is not matching.
